src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurSAEDetailLineFixture.java

```diff
@@ -189,13 +199,16 @@ private ConcurSAEDetailLineFixture(ConcurSAEDetailLineFixture baseFixture, Concu
                 overrideMap.getOrDefault(LineField.ORG_REF_ID, baseFixture.orgRefId),
                 journalDebitCredit,
                 journalAmount,
-                overrideMap.getOrDefault(LineField.REPORT_END_DATE, baseFixture.reportEndDate));
+                overrideMap.getOrDefault(LineField.REPORT_END_DATE, baseFixture.reportEndDate), 
+                overrideMap.getOrDefault(LineField.CASH_ADVANCE_KEY, baseFixture.cashAdvanceKey),
+                overrideMap.getOrDefault(LineField.REPORT_ENTRY_ID, baseFixture.reportEntryId));
     }
 
     private ConcurSAEDetailLineFixture(ConcurSAEFileFixture extractFile, ConcurEmployeeFixture employee,
             String reportId, String paymentCode, String chartOfAccountsCode,
             String journalAccountCode, String accountNumber, String subAccountNumber, String subObjectCode,
-            String projectCode, String orgRefId, String journalDebitCredit, double journalAmount, String reportEndDate) {
+            String projectCode, String orgRefId, String journalDebitCredit, double journalAmount, String reportEndDate,
+            String cashAdvanceCaKey, String reportEntryId) {

```

Comment:
It would be a good idea to rename the cash advance arg to be consistent with the "cashAdvanceKey" naming on the detail POJO.